content/post/mv-reflection-002.md

```diff
@@ -0,0 +1,44 @@
+
+And the the denouement, when Apple when public in 1980 a lot of the employees didn't get any shares and so...
+
+> **Livingston:** Didn’t you give away your Apple stock early on to other employees?
+>
+> **Wozniak:** As a matter of fact, when we went public, I was a little disturbed that five people who had been with us in our little office from the start and had been so important—Randy Wigginton, Chris Espinosa, a couple of young kids, and a couple of older ones, just hadn’t gotten any stock. I felt that they were a part of this whole energy and excitement and passion for what computers were going to be and what we were doing and how right it was. If somebody is sitting there working till 2:00 a.m. with you, helping to write a little code, and says, “Wow, that is a cool one,” those words mean a lot to you and they deserve something. So I gave each of those five a large amount of stock, probably a million dollars in that day. And that was an early day for a million dollars. I also did a program where I sold stock to about 40 Apple employees . . . I had a chance to sell some stock and get a house. There was an outside bigwig investor type that was willing to buy it all at a certain price. And I said, “Rather than sell it to somebody who’s already got a lot of money, why don’t I give the Apple employees the opportunity?” We were going to go public soon and it was going to be worth a lot more (and was eventually), so basically I sold it to 40 Apple employees. Our legal department was very concerned because they were supposed to be sophisticated investors. They finally gave me the OK. I did the deal and sold it to them, and they each pretty much got a house out of it.
+>
+> **Livingston:** That was so generous.
+>
+> **Wozniak:** But it’s that whole thing I was talking about: Hewlett-Packard, we’re a community. There was a recession in ’73 and Hewlett-Packard had to cut back 10 percent. Instead of laying off 10 percent of the people, they cut everyone’s salary by 10 percent and gave us one day off every two weeks. So basically they said “nobody goes without a job.” And I like that sort of thing. So a bunch of Apple engineers and marketing people got to benefit from going public. Otherwise, they’d have no stock at all. Mike Markkula kind of felt that some of these people didn’t deserve it; some people shouldn't get stock. But I disagreed with him on that. Nobody stopped me, so I did it.
+
+What struck me here is the inspiration for his generosity is not his vietnam-era politics, not some communitarian startup ethos, but the Hewlett-Packard corporate culture. And as a counterpoint, Apple the scrappy startup is already screwing over early employees 4 years into its existence.
```

Comment:
One of the things I'm also thinking about here, and this is related to your research question, is this idea that for real human relational endeavour, once you're operating right at the legal letter you've kind of already failed. The actual ethos of a relationship or a collaboration can not be fully formalized. This isn't to say that we shouldn't *have* contracts or rules, but once you're actually litigating the specifics of those that's a red flag. 

It seems that as they get larger organizations are more and more psychopathic. I can no longer trust that I looked someone in the eye and shook their hand and they will now do right by me.

So what's striking about Woz here is that he's operating like a human. He understands that there's of course a concept of fairness and justice that our contracts and legal systems are forever imperfect approximations of, and remembers that he has agency to act in a way that is more aligned with the underlying values. I actually see this ethic quite often in very nerdy/engineering types, which at first seems counterintuitive because the stereotype would be of a very analytical rules-based thinker, but of course the most creative engineers have a deep sense of the aesthetics of a system and can intuit how it *should* be built even when that doesn't match with some requirements document or the existing codebase. So to be a good engineer is exactly to tune into the underlying soul of a system. Here he's doing that for an organization, whereas Steve is trying to win some competition.

Reply:
> One of the things I'm also thinking about here, and this is related to your research question, is this idea that for real human relational endeavour, once you're operating right at the legal letter you've kind of already failed. The actual ethos of a relationship or a collaboration can not be fully formalized. This isn't to say that we shouldn't have contracts or rules, but once you're actually litigating the specifics of those that's a red flag.

I know why you're thinking about this Xavier ;)

Reply:
Jokes aside, I think your insight into Woz and his aesthetics around the soul of the organization are very thoughtful and a good place for me to think from.